### PR TITLE
[4294 : Resolved Tables overflow the content area if cell contains unbreakable text

### LINKED ISSUE #4294: Tables overflow the content area if cell contains unbreakable text
# Bug report

## Description

See https://devdocs.magento.com/guides/v2.3/extension-dev-guide/webapi/request-processor-
pool.html

This bug was introduced with https://github.com/magento/devdocs/pull/4105.
Since all our tables have `table-layout: auto` issues like this sometimes can happen.

### Steps to reproduce

Open the page with the table that has long lines:
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/webapi/request-processor-
pool.html

## Expected result

Table should not overflow the page content.

## Possible solutions

Make table scrollable?
Add `table-layout:fixed` by applying a `{: .fixed}` class to it.

### PR DESCRIPTION
**Pull request by hitesh-wagento**
## Purpose of this PR
Resolved Tables overflow the content area if cell contains unbreakable text

This PR ...

See https://devdocs.magento.com/guides/v2.3/extension-dev-guide/webapi/request-processor-pool.html
Since all our tables have table-layout: auto issues like this sometimes can happen.

## Affected URLs
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/webapi/request-processor-pool.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
Related to #4294.

@hitesh-wagento, it looks like you added a new class to the local CSS file. Our styling primarily comes from the https://github.com/magento-devdocs/devdocs-theme project. There's already a class there that we should review (as mentioned in the bug report: `table-layout:fixed`. See https://github.com/magento-devdocs/devdocs-theme/pull/34 for more info and work with @belbiy on changes.

File: common/css/devdocs.css [hunk 1]
```diff
@@ -1 +1 @@
-.home-contributors{margin-bottom:2em}.message-banner{margin:0}redoc[spec-url] table{table-layout:auto}redoc[spec-url] .menu-subitems .menu-item-title{text-transform:none}.language-bash.highlighter-rouge code:before{display:inline;content:'$';margin-right:5px;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.page-updated{margin:0 0 10px 0;color:#767676;cursor:help}.page-updated img{width:18px;height:18px}
+.home-contributors{margin-bottom:2em}.message-banner{margin:0}redoc[spec-url] table{table-layout:fixed}redoc[spec-url] .menu-subitems .menu-item-title{text-transform:none}.language-bash.highlighter-rouge code:before{display:inline;content:'$';margin-right:5px;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.page-updated{margin:0 0 10px 0;color:#767676;cursor:help}.page-updated img{width:18px;height:18px}.webapitable{table-layout:fixed}
```
Comment (osrecio) [inline common/css/devdocs.css]: Maybe table should remain with `table-layout:auto`

**Comment by belbiy** [comment]
We already have a class name `fixed` that you can apply to tables. 
Here's how to use it:

```
| Cell 1 | Cell 2 | Cell 3 |
{: .fixed }
```

Please do not add class names to the `devdocs.css` directly as well, since we're using npm to generate that file from `scss/devdocs.scss`

File: common/css/devdocs.css (hunk 1 above)
Comment (belbiy) [reply common/css/devdocs.css]: Do not change this file directly, please. This file is generated from `scss/devdocs.scss`. To generate it, use `npm run build:css`

File: guides/v2.2/extension-dev-guide/webapi/request-processor-pool.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ The request processors pool routes WebApi requests. It is located in the Magento
 
 Magento defines the following processors: 
 
+{:.webapitable}
```
[the remark this answers is not in the document]
Comment (hitesh-wagento) [reply guides/v2.2/extension-dev-guide/webapi/request-processor-pool.md]: Hi @belbiy 

I have added `.fixed` class in table as per your suggestion.

Thanks.

**Comment by hitesh-wagento** [comment]
Hi @belbiy , @osrecio 

I have added `.fixed` class in table as per your suggestion and removed static css.

Thanks.

File: common/css/devdocs.css (hunk 1 above)
Comment (hitesh-wagento) [reply common/css/devdocs.css]: Hi @belbiy 

Thanks for your suggestion. I have one confusion in  https://github.com/magento-devdocs/devdocs-theme Project.

Do I need to fork clone this repo in https://github.com/hitesh-wagento/devdocs or in new place?

Thanks for advance

**Comment by osrecio** [comment]
Hi @hitesh-wagento , I'm still seeing the css. Do you want that I delete it for you?

Thanks!

**Comment by hitesh-wagento** [comment]
> Hi @hitesh-wagento , I'm still seeing the css. Do you want that I delete it for you?
> 
> Thanks!

Hi @osrecio ,

I have revert that css before I have changed and added css. if you need remove that CSS you can do it.

Thanks.

**Pull request merged**